inference-engine/tests/functional/shared_test_classes/src/single_layer/convolution.cpp

```diff
@@ -41,23 +46,29 @@ std::string ConvolutionLayerTest::getTestCaseName(testing::TestParamInfo<convLay
 
 void ConvolutionLayerTest::SetUp() {
     convSpecificParams convParams;
-    std::vector<size_t> inputShape;
-    auto netPrecision   = InferenceEngine::Precision::UNSPECIFIED;
-    std::tie(convParams, netPrecision, inPrc, outPrc, inLayout, outLayout, inputShape, targetDevice) =
+    std::vector<std::pair<size_t, size_t>> inputShape;
+    std::vector<InferenceEngine::SizeVector> targetShapes;
+    std::tie(convParams, netPrecision, inPrc, outPrc, inLayout, outLayout, inputShape, targetShapes, targetDevice) =
         this->GetParam();
-    ngraph::op::PadType padType;
-    InferenceEngine::SizeVector kernel, stride, dilation;
-    std::vector<ptrdiff_t> padBegin, padEnd;
-    size_t convOutChannels;
+    for (auto&& targetShape : targetShapes) {
```

Comment:
I think, it is a bad idea to include `std::vector<InferenceEngine::SizeVector>` as a parameter of test. I think, it will be better to use only one shape in this case (`std::vector<size_t>`)

Reply:
I think that Irina asked using `std::vector<size_t>` not using `InferenceEngine::SizeVector`, right?